.docker/source/Dockerfile

```diff
@@ -1,35 +1,37 @@
+
+# syntax = docker/dockerfile:1.3
```

Comment:
What is this used for?

Reply:
This is required to allow mounting in the `RUN` command: 
https://github.com/ros-planning/moveit2/blob/399cc0302ca9749d6935839b5fb099bbe5a9fdd3/.docker/source/Dockerfile#L21